### PR TITLE
Follow X and Y coords when going to and from synctex

### PR DESCRIPTION
**Pull request by mpacer**
Working on this with @ian-r-rose.

### CODE REVIEW

**Comment by mpacer** [comment]
should we make it ⌘/⌃ + ⇧ + <click>? instead of ⌘/⌃ + <click>? That is what LaTeX tools does and I'm pretty sure that ⌃+<click> is supposed to be the equivalent of right clicking in windows.

I think @jasongrout had said something about this modifier + <click> issue before?

File: src/pdf.ts [hunk 1]
```diff
@@ -217,7 +217,16 @@ class PDFJSViewer extends Widget implements DocumentRegistry.IReadyWidget {
 
   private _handleClick(evt: MouseEvent): void {
     // If it is a normal click, return without doing anything.
-    if ((IS_MAC && !evt.metaKey) || (!IS_MAC && !evt.ctrlKey)) {
+    const checkKeys = (evt: MouseEvent): boolean => {
+      let shiftAccel: boolean;
+      if (IS_MAC) {
+        shiftAccel = evt.metaKey && evt.shiftKey;
```
Comment (ian-r-rose) [inline src/pdf.ts]: Why not return `evt.metaKey && evt.shiftKey`, and do away with `shiftAccel` altogether?

Comment (mpacer) [reply src/pdf.ts]: oh… because I was trying to avoid more than one return statement… which i'm now remembering is more the pattern used in javascript

Comment (ian-r-rose) [reply src/pdf.ts]: I don't have strong opinions, more of a personal preference thing.

**Comment by ian-r-rose** [comment]
:+1:

**Pull request merged**